docs/DeveloperGuide.md

```diff
@@ -158,37 +158,37 @@ This section describes some noteworthy details on how certain features are imple
 
 #### Proposed Implementation
 
-The proposed undo/redo mechanism is facilitated by `VersionedAddressBook`. It extends `AddressBook` with an undo/redo history, stored internally as an `addressBookStateList` and `currentStatePointer`. Additionally, it implements the following operations:
+The proposed undo/redo mechanism is facilitated by `VersionedSalesNote`. It extends `SalesNote` with an undo/redo history, stored internally as an `SalesNoteStateList` and `currentStatePointer`. Additionally, it implements the following operations:
 
-* `VersionedAddressBook#commit()` — Saves the current address book state in its history.
-* `VersionedAddressBook#undo()` — Restores the previous address book state from its history.
-* `VersionedAddressBook#redo()` — Restores a previously undone address book state from its history.
+* `VersionedSalesNote#commit()` — Saves the current address book state in its history.
+* `VersionedSalesNote#undo()` — Restores the previous address book state from its history.
+* `VersionedSalesNote#redo()` — Restores a previously undone address book state from its history.
```

Comment:
Appreciate that you went through the hard effort of fixing this entire section to reflect the addressbook. I think we could have removed this section entirely actually, since we dont even know if we will implement the undo/redo. Since you already did this, i'm happy to keep it though!